types/jest/index.d.ts

```diff
@@ -308,10 +327,43 @@ declare namespace jest {
      */
     function useRealTimers(): typeof jest;
 
-    interface MockOptions {
-        virtual?: boolean | undefined;
+    interface MockOptions { // tslint:disable-next-line: no-trailing-whitespace
+        virtual?: boolean | undefined; // tslint:disable-next-line: no-trailing-whitespace
+    }    
```

Comment:
```suggestion
    interface MockOptions {
        virtual?: boolean | undefined;
    }
```

There's no reason to keep trailing whitespace.

Reply:
earlier CI keeps failing at these lines idk why (may be due to prettier changes)  ? but now its look fine .